Contents/Code/__init__.py

```diff
@@ -163,14 +163,17 @@ def Update(self, metadata, media, lang, force, movie):
 
         if not movie:
             for ep in series['eps']:
-                if ep['eptype'] != "Episode" and ep['eptype'] != "Special":
+                if ep['eptype'] != "Episode" and ep['eptype'] != "Special" and ep['eptype'] != "Credits" and ep['eptype'] != "Trailer":
```

Comment:
This might be better written as `not (ep['eptype'] in ["Episode", "Special", "Credits", "Trailer"]) `